core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -34,12 +34,65 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     case ir.DeleteFromTable(target, None, where, None, None) => delete(ctx, target, where)
     case c: ir.CreateTableCommand => createTable(ctx, c)
     case t: ir.TableSample => tableSample(ctx, t)
+    case a: ir.AlterTableCommand => alterTable(ctx, a)
     case ir.NoopNode => ""
     case ir.UnresolvedCommand(inputText) => s"--${inputText}"
     case null => "" // don't fail transpilation if the plan is null
     case x => throw unknown(x)
   }
 
+  private def alterTable(ctx: GeneratorContext, a: ir.AlterTableCommand): String = {
+    val operation = buildAlter(ctx, a.alterations)
+    s"ALTER TABLE ${a.tableName} $operation"
+  }
+
+  private def buildAlter(ctx: GeneratorContext, alteration: Seq[ir.TableAlteration]): String = {
+    alteration match {
+      case a: Seq[_] => buildTableAlteration(ctx, a)
+      case _ => ""
+    }
+  }
+
+  private def buildTableAlteration(ctx: GeneratorContext, c: Seq[_]): String = {
+
+    c match {
+      case a: Seq[_] =>
```

Comment:
`case a: Seq[_] =>` is bad code and it's not clear what it does.

Reply:
Fixed it